src/Data/Managers/V2/GetDesiredStates/getUrl.ts

```diff
@@ -0,0 +1,53 @@
+import moment from "moment-timezone";
+import qs from "qs";
+import { Query, RangeOperator } from "@/Core";
+
+/**
+ * Constructs the URL for fetching desired states based on the provided query parameters.
+ * @note This function is imported from V1 of Query Manager of the same url query manager, as filters are out of scope of the query manager upgrade/migration
```

Comment:
Can you make a ticket to update the getUrl methods to works with React Query, and link it in the comment?